### PR TITLE
Added RTD link to Readme

### PR DESCRIPTION
**Pull request by Ayushsunny**
Link to RTD documentation from README 
issue code - #915

### CODE REVIEW

**Comment by Ayushsunny** [comment]
@vringar Could you review it sir? Please

**Comment by vringar** [comment]
Hey,
this PR looks good. However I'd prefer it if the Documentation paragraph was further up. 
Could you please move it between the Troubleshooting and Advice for Measurement Researchers?

**Comment by Ayushsunny** [comment]
> Hey,
> this PR looks good. However I'd prefer it if the Documentation paragraph was further up.
> Could you please move it between the Troubleshooting and Advice for Measurement Researchers?

Sure Sir,
I have amend the changes.

File: README.md [hunk 1]
```diff
@@ -31,6 +32,7 @@ Table of Contents <!-- omit in toc -->
   - [MacOS GUI applications in Docker](#macos-gui-applications-in-docker)
 - [Citation](#citation)
 - [License](#license)
+- [Documentation](#documentation)
```
Comment (vringar) [inline README.md:35]: Please also move this up in the table of contents

**Review by vringar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ayushsunny) [reply README.md]: Done 👍🏻

**Review by vringar — APPROVED** [review]

**Comment by Ayushsunny** [comment]
Hello @vringar is there any other issue's present now ? or something I can work on.

**Pull request merged**